onnxruntime/core/optimizer/selectors_actions/selector_action_transformer.cc

```diff
@@ -54,18 +55,22 @@ const SelectorActionRegistry::Entry* SelectorActionRegistry::LookUp(const std::s
 }
 
 #if !defined(ORT_MINIMAL_BUILD)
-const SelectorActionRegistry::Entry* SelectorActionRegistry::LookUpByOpType(const std::string& op_type) const {
-  if (const auto it = op_type_to_entry_.find(op_type); it != op_type_to_entry_.end()) {
-    return it->second;
-  }
-  return nullptr;
+auto SelectorActionRegistry::LookUpByOpType(const std::string& op_type) const
```

Comment:
What's the benefit to using 'auto' as the return type and defining the return type after the `->`?

Reply:
this way `Entry` can be used instead of `SelectorActionRegistry::Entry`